### DISCUSSION TITLE
64 GB RAM | Issues

### DISCUSSION THREAD

**Discussion by haneiter** [osy86/HaC-Mini#1012] [General]
Hi,

first of thank you for the guide and the work. I just finished yesterday with a fresh install. 
12.4 Monterey works great so far, everything works (changed WiFi Card) - even have Win 10 on a second SSD.

I had some spare 2x32 GB No Name PC4-3200 RAM and wanted to try to see how the performance is.
It boots up and works but I get random freezes. 

Deactivated the Second SSD in BIOS to rule out that this is the issue. Still got some random Freezes.
Most of the Time "Windows Server" is crashing. Hard to replicate the issue.

I switched back to the old 32GB Sticks and since then its stable. 

Any advice if/how I can make 64 GB work or how to troubleshoot the issue?

Edit: Here are more details for the used RAM.
![51656849377_ pic](https://user-images.githubusercontent.com/108607900/177044689-a73c0f03-e223-4aed-85c8-7e1025c495d5.jpg)

Tried benchmarking with Cinebench. 32GB runs normal 30min stability test. 64 crashes :/

**Comment by Medizinmann-MD** [comment]
I run 64 GB using 2 stick 32 GB Crucial Balistix CL16 3200 MT/s with no issues for 2 years now.

I would guess it is the XMP-Profile of your no name RAM - maybe you need to bump the Voltage by 0,02-0,05 V or something - this at least what almost always helped with failing sticks in the desktop machines I assembled.

**Comment by haneiter** [comment]
Hi,

your answer helped a lot. Was not aware of XMP Profiles. When set to 2667 instead of 3200 it was stable.
Proceeded to dive deeper and ran memtest. Found 6 Errors

Swapping them for error free ones took nearly 2 Weeks. Running 64GB at 3200 now and and so far no issues.

If I need to switch to 2667 again I will update. But seems like Timings and/or fine-tuning XMP Profiles seems to be a good way to troubleshoot!

**Reply by Medizinmann-MD** [reply]
> Hi,
> 
> your answer helped a lot. Was not aware of XMP Profiles. When set to 2667 instead of 3200 it was stable. Proceeded to dive deeper and ran memtest. Found 6 Errors
> 
> Swapping them for error free ones took nearly 2 Weeks. Running 64GB at 3200 now and and so far no issues.
> 
> If I need to switch to 2667 again I will update. But seems like Timings and/or fine-tuning XMP Profiles seems to be a good way to troubleshoot!

If you don't touch the RAM-Sticks it is rather unlikely that they will fail again - usually they are DOA or dead by static.

**Comment by Medizinmann-MD** [comment]
I can report that Mushkin Redline SO-DIMM Kit 64GB, DDR4-3200, CL16-18-18-38 also works perfectly well.
